### PR TITLE
Support view rebase

### PR DESCRIPTION
**Pull request by kernelkind**
rebased support view to have features requested in #379 and is on top of #360

### CODE REVIEW

**Comment by jb55** [comment]
This broke something, you can't do -c notifications anymore

**Comment by jb55** [comment]
but -c notifications:npub... works

**Comment by kernelkind** [comment]
> This broke something, you can't do -c notifications anymore

that's weird, I can do `-c notifications` just fine...

**Comment by jb55** [comment]
with --sec accounts?

--sec nsec -c notifications

File: src/app.rs [hunk 1]
```diff
@@ -694,6 +694,7 @@ impl Damus {
         for key in parsed_args.keys {
             info!("adding account: {}", key.pubkey);
             accounts.add_account(key);
+            accounts.select_account(accounts.num_accounts() - 1);
```
Comment (jb55) [inline src/app.rs:697]: should be first account

Comment (jb55) [reply src/app.rs:697]: ill fix

**Comment by jb55** [comment]
thanks, still one issue with the `open` action:

> when I do `open 'mailto:jb55@jb55.com'` on my commandline I get mail app. I don't think our function is doing the right thing. in notedeck its always opening my browser, but I don't have a browser mailto handler

**Pull request merged**